src/farmers/farmerbot/farmerbot_quick.md

```diff
@@ -40,76 +64,210 @@ Here are some examples to guide you:
-* Pull the file creator image from the GitHub Container Registry
-  * ```
-    docker pull ghcr.io/threefoldtech/farmerbot_config:0.2.0
+* Set the Farmerbot systemd service
+    
     ```
-* Run the file creator image and answer the questions to generate the files
-  * ```
-    docker run --name fbot_config_container -v ./:/farmerbot -ti ghcr.io/threefoldtech/farmerbot_config:0.2.0
+    [Unit]
+    Description=ThreeFold Farmerbot
+    StartLimitIntervalSec=0
+
+    [Service]
+    Restart=always
+    RestartSec=5
+    StandardOutput=append:/root/farmerbotfiles/farmerbot.log
+    StandardError=append:/root/farmerbotfiles/farmerbot.log
+    ExecStart=/usr/local/bin/farmerbot run -e /root/farmerbotfiles/.env -c /root/farmerbotfiles/config.yml -d
+
+    [Install]
+    WantedBy=multi-user.target     
+    ```
+* Enable the Farmerbot service
+    ```
+    systemctl daemon-reload
+    systemctl enable farmerbot
+    systemctl start farmerbot
+    ```
+* Verify that the Farmerbot service is properly running
+    ```
+    systemctl status farmerbot
     ```
-    * The `config/config.md` and `.env` files are saved in the parent folder
 
-> Note: It is can be a good idea to set only one node with `yes` for `never_shutdown`. A node with low power consumption is obviously a wise choice.
+### Check the Farmerbot Logs
 
+Once you've set a Farmerbot systemd service [as show above](#set-a-systemd-service), the Farmerbot will start writing logs to the file `farmerbot.log` in the directory `farmerbotfiles`.
 
+Thus, you can get more details on the operation of the Farmerbot by inspecting the log file. This can also be used to see the **Farmerbot Report Table** as this table is printed in the Farmerbot log.
 
-## Run the Farmerbot
+* See all logs so far
+  ```
+  cat ~/farmerbotfiles/farmerbot.log
+  ```
+* See the last ten lines and new logs as they are generated
+  ```
+  tail -f ~/farmerbotfiles/farmerbot.log
+  ```
+* See all logs and new lines as they are generated
+  ```
+  tail -f -n +1 ~/farmerbotfiles/farmerbot.log
+  ```
+* See the last report table
+  ```
+  tac farmerbot.log | grep -B5000 -m1 "Nodes report" | tac
+  ```
 
-* Download the Farmerbot `.yaml` file with this command:
-  * ```
-    wget https://raw.githubusercontent.com/threefoldtech/farmerbot/development/docker-compose.yaml
-    ```
-* Run the Farmerbot with this command
-  * ```
-    docker compose up -d
-    ```
+## Farmerbot Files
+
+### Configuration File Template (config.yml)
+
+In this example, the farm ID is 1, we are setting the Farmerbot with 4 nodes and the node 1 never shuts down, we set a periodic wakeup at 1:00PM.
+
+Note that the timezone of the farmerbot will be the same as the time zone of the machine the farmerbot running inside. By default, a full VM on the TFGrid will be set in UTC.
+
+```
+farm_id: 1
+included_nodes:
+  - 1
+  - 2 
+  - 3 
+  - 4  
+never_shutdown_nodes:
+  - 1
+power:
+  periodic_wake_up_start: 01:00PM
+```
```

Comment:
Can you also add `excluded_nodes` and its description?

And also,
power:
  periodic_wake_up_start: 
  wake_up_threshold:
  periodic_wake_up_limit: 
  overprovision_cpu:

All the details are [here](https://github.com/threefoldtech/tfgrid-sdk-go/blob/development/farmerbot/README.md#config).

Reply:
Yes good catch. This info was present on the **Additional Information** page with a link from farmerbot_quick page, but it was removed inadvertently in [this commit](https://github.com/threefoldtech/info_grid/commit/9bcabfe0f1aa92cba9aff015494dbde472d38291).

Will fix that and update any new info if needed, as well as work on your other suggestions above. Thanks @A-Harby!

Reply:
Done. I refer to the additional section where all info is there. On the quick guide we give the basic example.